packages/suite/src/views/wallet/send/components/Fees/components/CustomFee/index.tsx

```diff
@@ -63,12 +63,10 @@ const CustomFee = () => {
                             return 'CUSTOM_FEE_IS_NOT_NUMBER';
                         }
                         // allow decimals in ETH since GWEI is not a satoshi
-                        if (
-                            network.networkType !== 'ethereum' &&
-                            (!feeBig.isInteger() || value.includes('.'))
-                        ) {
+                        if (network.networkType !== 'ethereum' && !isInteger(value)) {
```

Comment:
Not sure if we have the network types in a constant file but I'd be useful to create one in the near future.

Reply:
we do have it since early beginnings:
https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/config/wallet/networks.ts